src/lib.rs

```diff
@@ -336,6 +336,6 @@ pub mod text;
 pub mod widgets;
 
 #[doc(inline)]
-pub use self::terminal::{CompletedFrame, Frame, Terminal, TerminalOptions, Viewport};
+pub use crate::terminal::{CompletedFrame, Frame, Terminal, TerminalOptions, Viewport};
```

Comment:
This also works. Could be less or more confusing, depending on who you ask.

```suggestion
pub use terminal::{CompletedFrame, Frame, Terminal, TerminalOptions, Viewport};
```

Reply:
I think I'd like to normalize this to something, but it's lower priority. The self seemed odd in a few places though. I don't have strong feelings about it though.

Reply:
1051d9db1